router/src/main/kotlin/io/moia/router/Router.kt

```diff
@@ -77,4 +77,6 @@ data class Request<I>(val apiRequest: APIGatewayProxyRequestEvent, val body: I,
 
     val pathParameters by lazy { UriTemplate.from(pathPattern).extract(apiRequest.path) }
     fun getPathParameter(name: String): String? = pathParameters[name]
+    fun getQueryParameter(name: String): String? = apiRequest.queryStringParameters?.get(name)
```

Comment:
Do we want to add `val queryParameters` and `val multiValueQueryParameters` as well? (Since we would otherwise still use the raw `.apiRequest` in some places)

Reply:
Why not. Added it.